docs/installation.md

```diff
@@ -25,6 +25,16 @@ The Lychee gallery has a few system requirements. You will need to make sure you
    		- Tokenizer
    		- XML
    		- ZIP
+   	- These PHP extensions are necessary if you are running a FreeBSD system:
```

Comment:
I'd drop this line and merge the lists. Other extensions (e.g. JSON) are included by default, so we should probably list all of them.